tests/testMeasureSources.py

```diff
@@ -295,7 +291,7 @@ def testPixelFlags(self):
         source.set("centroid_y", 40)
         source.set("centroid_flag", True)
         source.setFootprint(afwDetection.Footprint(afwGeom.Point2I(afwGeom.Point2D(x + x0, y + y0)), 5))
-        self.assertRaises(  
+        self.assertRaises(
```

Comment:
@r-owen has suggested we rewrite these using:
```python
with self.assertRaises(lsst.pex.exceptions.RuntimeError):
    plugin.measure(source, exp)
```
I agree with him that this looks much more readable and obvious.